src/io/driverominit.c

```diff
@@ -1,31 +0,0 @@
-#include "PR/os_internal.h"
```

Comment:
I'm not sure removing these DD functions belongs in this PR

Reply:
The way I saw it these DD functions were an ultralib leftover, because of how ultralib removes files from the final libraries using predefined lists per version
Because hackerlibultra doesn't do this, code from older libultra versions removed in newer versions still exists in the repository

Reply:
The reasoning makes sense, I just don't think it belongs in a PR focused on removing indy/debug code. Could the DD functions be removed in a separate PR?

Reply:
Files reinstated